osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -40,19 +40,27 @@ func addCreateOsConfigTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	desc := "test osconfig creation"
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
+		var image string
 
 		switch pkgManager {
 		case "apt":
+			image = debianImage
 			pkg := osconfigserver.BuildPackage("cowsay")
 			pkgs := []*osconfigpb.Package{pkg}
-			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s", filepath.Base(image), testName), desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
+			break
+		case "yum":
+			image = centosImage
+			pkg := osconfigserver.BuildPackage("cowsay")
+			pkgs := []*osconfigpb.Package{pkg}
+			oc = osconfigserver.BuildOsConfig(fmt.Sprintf("%s-%s", filepath.Base(image), testName), desc, nil, osconfigserver.BuildYumPackageConfig(pkgs, nil, nil), nil, nil, nil)
 			break
 		default:
 			panic(fmt.Sprintf("non existent package manager: %s", pkgManager))
 		}
 		setup := packageManagementTestSetup{
-			image:      debianImage,
-			name:       fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName),
+			image:      image,
+			name:       fmt.Sprintf("%s-%s", filepath.Base(image), testName),
```

Comment:
path.Base here as this is a URI

Reply:
doen... and changed at other places as well